docs/proposals/coap-core.md

```diff
@@ -0,0 +1,74 @@
+# [CoAPCore](https://tools.ietf.org/html/rfc6690#:~:text=well-known%2Fcore) Protocol Implementation
```

Comment:
this is nit but also a helpful clarification to me as I learn about CoRE, but is this better named CoAP CoRE?

Reply:
Do you mean if there's a better name than CoAP CoRE? Officially the discovery protocol is CoRE, but the implementation is tailored for CoRE under CoAP. As the RFC6690 states, CoRE can be applied also to other protocols like HTTP, so it could be named only CoRE.

For the time being I chose CoAP CoRE to be explicit about the expected communication protocol used by the devices. But I can imagine in future that the same discovery handler could support also HTTP or even MQTT etc.

Reply:
Cool. I was specifically referring to changing Core to CoRE, but this has reinforced the fact that I need to learn more about CoRE and how it can be applied to other protocols.

Reply:
I'll correct the name to `CoAP CoRE` in the proposal title 👍